00-RELEASENOTES

```diff
@@ -9,7 +9,7 @@ Usually "unstable" is stable enough for you to use it in development environment
 however you should never use it in production environments. It is possible
 to download the latest stable release here:
 
-    http://download.redis.io/releases/redis-stable.tar.gz
+    https://download.redis.io/releases/redis-stable.tar.gz
```

Comment:
This isn't a valid address, not sure why we stopped populating it.

Reply:
i see it is currently here:
https://download.redis.io/redis-stable.tar.gz
the deploy scripts i got from Salvatore seem to put it there, so i'm not sure if it was ever present in the `releases` folder.
@itamarhaber do you know something?

Reply:
I'm of the same opinion - stable was always at the root, numbered releases under the 'releases' folder.

Should be fixed in future releases of the release-notes.